go-controller/pkg/cni/ovs.go

```diff
@@ -66,3 +68,30 @@ func ovsClear(table, record string, columns ...string) error {
 	_, err := ovsExec(args...)
 	return err
 }
+
+func ofctlExec(args ...string) (string, error) {
+	args = append([]string{"--timeout=30"}, args...)
+	output, err := runner.Command("ovs-ofctl", args...).CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("failed to run 'ovs-ofctl %s': %v\n  %q", strings.Join(args, " "), err, string(output))
+	}
+
+	outStr := string(output)
+	trimmed := strings.TrimSpace(outStr)
+	// If output is a single line, strip the trailing newline
+	if strings.Count(trimmed, "\n") == 0 {
+		outStr = trimmed
+	}
+
+	return outStr, nil
+}
+
+func waitForBrIntFlows(ip string) error {
+	return wait.PollImmediate(100*time.Millisecond, 20*time.Second, func() (bool, error) {
+		stdout, err := ofctlExec("dump-flows", "br-int", fmt.Sprintf("table=9,ip,nw_src=%s", ip))
```

Comment:
`ip` and `nw_src` both mean IPv4 only. You need to check `utilnet.IsIPv6String(ip)` and use `ipv6,ipv6_src=%s` in that case

Reply:
done, or will use the fix we end up with in downstream: https://github.com/openshift/ovn-kubernetes/pull/226